ui/package.json

```diff
@@ -58,10 +58,10 @@
         "emotion": "^10.0.27",
         "express": "^4.16.4",
         "fetchr": "^0.5.38",
-        "flatpickr": "^4.6.3",
+        "flatpickr": "4.6.3",
```

Comment:
- Ran ynpm-check --security to check for S1 vulnerabilities and fixed them. 
- Wen't over the release notes of each package and verified the breaking changes didn't affect us.
- Ran a local version of the UI and did a short sanity test (created a role, added members, viewed history). Everything seems in order